### PR TITLE
Populate PeriodType appropriately

### PR DESCRIPTION
**Pull request by brancz**
PeriodType is intended to explain the kind of events between sampled
occurrences. While not strictly required to be set, all the Go stdlib
profilers populate it, and this patch sets it equivalent to the standard
CPU profiles.

Resolves part of #16

### CODE REVIEW

**Review by kakkoyun — APPROVED** [review]

File: format.go [hunk 1]
```diff
@@ -60,6 +60,10 @@ func toPprof(s map[string]int, hz int) *profile.Profile {
 			Unit: "nanoseconds",
 		},
 	}
+	p.PeriodType = &profile.ValueType{
+		Type: "cpu",
```
Comment (felixge) [inline format.go:64]: I think this is misleading. fgprof does not produce CPU profiles. It produces "wallclock" profiles.

So maybe this?

```suggestion
		Type: "wallclock",
```

**Review by felixge — CHANGES_REQUESTED** [review]
Thanks for the patch and sorry for the slow response. PTAL at my comment.

File: format.go (hunk 1 above)
Comment (brancz) [reply format.go]: Fixed.

**Review by bwplotka — APPROVED** [review]
Looking forward for this, thanks for fix @brancz 👍🏽

**Comment by felixge** [comment]
Thanks for the patch. Sorry for the slow response. Noticed some accidental code duplication while reviewing this, so pushed a small fix for that before merging.

**Pull request merged**